pbm/storage/s3/s3.go

```diff
@@ -478,10 +480,18 @@ func (s *S3) session() (*session.Session, error) {
 	return session.NewSession(&aws.Config{
 		Region:   aws.String(s.opts.Region),
 		Endpoint: aws.String(s.opts.EndpointURL),
-		Credentials: credentials.NewStaticCredentials(
-			s.opts.Credentials.AccessKeyID,
-			s.opts.Credentials.SecretAccessKey,
-			"",
+		Credentials: credentials.NewChainCredentials(
+			[]credentials.Provider{
+				&credentials.EnvProvider{},
+				&ec2rolecreds.EC2RoleProvider{
+					Client: ec2metadata.New(session.New()),
+				},
+				&credentials.StaticProvider{Value: credentials.Value{
```

Comment:
I think it's worth changing the order of credentials search, since if credentials are explicitly set in config file, the defined creds should take priority over automatically provided access.

@dAdAbird, what do you think?

Reply:
yes, I think you are right, PBM config values should be the first priority. 
@eddycharly can you address this?

Reply:
Sure, i'll fix that tomorrow.

Reply:
@dAdAbird @sandraromanchenko the order of providers was changed according to your feedback.

Reply:
LGTM)

Reply:
Thanks @sandraromanchenko :)
What are the next steps ? Are you releasing new versions regularly ?